modules/dnn/test/test_darknet_importer.cpp

```diff
@@ -125,13 +125,20 @@ class Test_Darknet_nets : public DNNTestLayer
                           const std::vector<int>& refClassIds,
                           const std::vector<float>& refConfidences,
                           const std::vector<Rect2d>& refBoxes,
-                          double scoreDiff, double iouDiff, float confThreshold = 0.24)
+                          double scoreDiff, double iouDiff, float confThreshold = 0.24, int batch_size = 1)
     {
         checkBackend();
 
         Mat sample = imread(_tf("dog416.png"));
-        Mat inp = blobFromImage(sample, 1.0/255, Size(416, 416), Scalar(), true, false);
-
+        Mat inp;
+        if (batch_size > 1){
+            // batch_size matrices with value sample
+            std::vector<Mat> samples(batch_size, sample);
+            inp = blobFromImages(samples, 1.0/255, Size(416, 416), Scalar(), true, false);
+        }
+        else{
+            inp = blobFromImage(sample, 1.0/255, Size(416, 416), Scalar(), true, false);
+        }
```

Comment:
You can replace all these lines to the following ones:
```cpp
std::vector<Mat> samples(batch_size, sample);
Mat inp = blobFromImages(samples, 1.0/255, Size(416, 416), Scalar(), true, false);
```
For `batch_size == 1` it works in the same way as
```cpp
blobFromImage(sample, 1.0/255, Size(416, 416), Scalar(), true, false);
```